psi4/src/psi4/libsapt_solver/usapt0.cc

```diff
@@ -2340,17 +2340,17 @@ void USAPT0::mp2_terms()
             int nb_bblock = (bstart + maxb_b >= nbb ? nbb - bstart : maxb_b);
 
             if ( na_bblock > 0) {
-                fread(Aa_bsp[0],sizeof(double),na_bblock*nasQ,Aa_bsf);
-                fread(Ba_brp[0],sizeof(double),na_bblock*narQ,Ba_brf);
-                fread(Ca_brp[0],sizeof(double),na_bblock*narQ,Ca_brf);
-                fread(Da_bsp[0],sizeof(double),na_bblock*nasQ,Da_bsf);
+                size_t val=fread(Aa_bsp[0],sizeof(double),na_bblock*nasQ,Aa_bsf);
+                val=fread(Ba_brp[0],sizeof(double),na_bblock*narQ,Ba_brf);
+                val=fread(Ca_brp[0],sizeof(double),na_bblock*narQ,Ca_brf);
+                val=fread(Da_bsp[0],sizeof(double),na_bblock*nasQ,Da_bsf);
```

Comment:
This looks like a bugfix where you don't actually care about `val`. What's the bug, that I may recognize it :-)